Cargo.toml

```diff
@@ -20,10 +20,12 @@ exclude = ["tests/images/*", "tests/fuzz_images/*"]
 half = { version = "2.4.1" }
 quick-error = "2.0.1"
 
+fax = { version = "0.2.4", optional = true }
```

Comment:
nit: Please add a "fax" feature below and gate both fax and bitvec on it.

Reply:
I've gotten rid of `bitvec` (quite some dependencies and we only use its `Extend`…) and added a feature, renaming the dependency in the process to avoid confusion.